server/src/main/java/com/hedera/block/server/health/HealthServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.server.health;
+
+import com.hedera.block.server.ServiceStatus;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.helidon.webserver.http.HttpRules;
+import io.helidon.webserver.http.ServerRequest;
+import io.helidon.webserver.http.ServerResponse;
+
+/** Provides implementation for the health endpoints of the server. */
+public class HealthServiceImpl implements HealthService {
+
+    private static final String LIVENESS_PATH = "/liveness";
+    private static final String READINESS_PATH = "/readiness";
+
+    private final ServiceStatus serviceStatus;
+
+    /**
+     * It initializes the HealthService with needed dependencies.
+     *
+     * @param serviceStatus is used to check the status of the service
+     */
+    public HealthServiceImpl(@NonNull ServiceStatus serviceStatus) {
+        this.serviceStatus = serviceStatus;
+    }
+
+    @Override
+    public String getHealthRootPath() {
+        return "/health";
```

Comment:
I think the K8s convention is `healthz` https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request

Reply:
I don't know why I don't like the `z` but changed it already so is more standard.

✅  Done!